packages/backend/src/app.ts

```diff
@@ -462,4 +500,52 @@ export class App {
           ).unref()
       })
   }
+
+  private isDatabaseRowExpired<TRecord, TResult>(
+    now: number,
+    row: QueryInterface<TRecord, TResult>,
+    rule: DatabaseCleanupRule
+  ): boolean {
+    // get the base time used to compute the row's age
+    let absoluteLapseTime = row.column[rule.absoluteStartTimeColumnName]
+
+    if (rule.lapseTime) {
+      const timeParameter =
+        row.column[rule.lapseTime.columnName] || rule.minLapseTimeMillis
+      if (rule.lapseTime.absolute) {
+        absoluteLapseTime = timeParameter
+      } else {
+        // offset the working base time by a time span stored in another column
+        absoluteLapseTime += timeParameter
```

Comment:
Interesting that this works given that one is a timestamp and one is an integer.

Reply:
added `.valueOf()` just to be safe